vertical-pod-autoscaler/pkg/admission-controller/resource/vpa/handler_test.go

```diff
@@ -150,6 +152,50 @@ func TestValidateVPA(t *testing.T) {
 			},
 			expectError: fmt.Errorf("max resource for cpu is lower than min"),
 		},
+		{
+			name: "bad minAllowed cpu value",
+			vpa: vpa_types.VerticalPodAutoscaler{
+				Spec: vpa_types.VerticalPodAutoscalerSpec{
+					ResourcePolicy: &vpa_types.PodResourcePolicy{
+						ContainerPolicies: []vpa_types.ContainerResourcePolicy{
+							{
+								ContainerName: "loot box",
+								MinAllowed: apiv1.ResourceList{
+									cpu: badCPUValue,
+								},
+								MaxAllowed: apiv1.ResourceList{
+									cpu: resource.MustParse("275m"),
```

Comment:
`275m` is a whoe number of milli CPUs

Reply:
This test case will exit due to memory minAllow invalidate(100m)  and will not go to the maxcpu validation